tasks/reproducibility-xfel.tex

```diff
@@ -3,42 +3,165 @@
+  analysis commands with outputs, and that the notebook provides a
+  complete (and thus reproducible) summary of the data analysis when
+  it succeeds with the execution. Should the execution fail, for
+  example half-way through the notebook, then derived results obtained
+  prior to the error occurring are preserved and can be inspected. The
+  error is embedded in the notebook and appears after the command that
+  has triggered the error; which helps with debugging the process.
+
+  This is of particular interest as the data analysis processes at
+  European XFEL may fail not because of software errors but due to
+  variation in the data that require (manual) expert adjustments of
+  parameters. The ``failure'' of such an analysis workflow
+  (represented through the Notebook) is thus not exceptional, but a
+  common occurrence. The scientist conducting the experiment is
+  sufficiently skilled to modify the parameters and wants to either
+  re-execute the notebook from the beginning or to continue from the
+  point of failure. The notebook caters for both use cases. The
+  modified notebook would need to be preserved of course to provide
+  reproducibility of the derived results that the notebook has
+  computed.
+
+  We are aiming for re-executability of the notebook for the lifetime
+  of the data. The life time of the archived data at European XFEL is
+  currently guaranteed for 5 years and aimed to be 10 years
+  \cite{EuXFEL-datapolicy-2017}. It is possible though, that data used
+  for publications will be preserved for longer, and it would be
+  highly desirable to keep the data analysis re-executable for the
+  same period of time, potentially well exceeding 10 years.
+
+
+
+%%%
+%%% A major category of experiments already begun at European XFEL revolve around
+%%% X-ray crystallography, using X-ray scattering to resolve molecular structures.
+%%% Processing the data collected to produce meaningful structures is a
+%%% significant computational task, and researchers have developed complex
+%%% software for this, such as CrystFEL, Cheetah and CORTEX.
+%%%
+%%% As an application of the reproducible workflows task in WP2, we will build
+%%% workflows around such software, aiming to allow 'one button'' replication
+%%% of published structures once the raw data is made available.
+%%% As part of European XFEL's data policy, all data is to be publicly available
+%%% ?? years after it is collected.
+
+  \medskip \emph{This task} includes the following activities:
   \begin{compactitem}
-  \item ...
-    (\localdelivref{deliv-id})
+  \item Aim to use the software archive for reproducible computation
+    (as co-developed in \taskref{ecosystem}{reproducibility}), to
+    provide reproducible computation environments for data analysis at
+    European XFEL that remains executable for the same duration as the
+    data is kept (currently aiming at 10 years, at least 5).
+
+    As often in computational science, software used at XFEL is often
```

Comment:
'Often' appears twice close together. Maybe replace the first one with 'As is common in...'

I also tend to refer to 'scientific computing', because 'computational science' sounds a lot like 'computer science'. But I think it's clear which one we mean, and perhaps this is more a personal preference.